### DISCUSSION TITLE
Unable to get to button

### DISCUSSION THREAD

**Discussion by amitlzkpa** [simple-login/app#1809] [Q&A] [answered]
Hi
I tried to add the SimpleLogin add-on but can't see the button as shown on the onboarding page.
I see the Firefox relay button.

![image](https://github.com/simple-login/app/assets/15354742/9b8014ab-ee0d-42b8-87b4-3d41d5c63a43)

**Comment by nguyenkims** [comment, marked as answer]
It's because both extensions want to display their icon at the same position. You can either uninstall Firefox Relay or enable this option on SL
<img width="433" alt="image" src="https://github.com/simple-login/app/assets/3167780/f2e2cd24-35c7-447b-9834-997410a1b090">
